modules/optimeraRtdProvider.js

```diff
@@ -34,6 +34,8 @@ export let optimeraKeyName = 'hb_deal_optimera';
 
 /**
  * Optimera Score File Base URL.
+ * This is the base URL for the data endpoint request to fetch
+ * the targeting values.
  * @type {string}
```

Comment:
Why does it say .js at the end when you construct the url from pieces of it is a data url